presto-docs/src/main/sphinx/orm/list.rst

```diff
@@ -0,0 +1,17 @@
+=====================
+ORM
+=====================
+
+.. contents::
+    :local:
+    :backlinks: none
+    :depth: 1
+
+Overview
+--------
+
```

Comment:
Could you add a sentence or short paragraph introducing the concept of ORM, expanding the words that ORM represents, and why ORM can be useful when working with Presto? This doesn't have to be and shouldn't be long, but you can help the user by adding a little context.

Reply:
removed the category `Orm`